getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Create New Project
+page_title: Create New Project - Visual Studio Code Integration
+description: Learn how to create a new Telerik UI for Blazor project with our Visual Studio Code Templates.
+slug: getting-started-vs-code-integration-new-project
+position: 2
+---
+
+
+# Create New Projects
+
+This article demonstrates how to create a new application that has the Progress&reg; Telerik&reg; UI for Blazor components working.
```

Comment:
Add a couple of words to denote that this is to create a new project wit hthe telerik components **in VS Code**

Reply:
Done.